tests/Behat/Context/Ui/RefundingContext.php

```diff
@@ -76,6 +76,18 @@ public function decideToRefundProductWithVeryLongComment(
         $this->orderRefundsPage->refund();
     }
 
+    /**
+     * @When /^I decide to refund (\d+) "([^"]*)" products with "([^"]*)" payment$/
+     */
+    public function iDecideToRefundProductsWithPayment(int $amout, string $productName, string $paymentMethod): void
+    {
+        for($number = 0;$number<$amout;$number++) {
```

Comment:
Give them some space :) 

```suggestion
        for($number = 0; $number < $amout; $number++) {
```